Extensions/ArtifactEngine/E2ETests/E2E.ts

```diff
@@ -89,8 +89,36 @@ describe('e2e tests', () => {
             });
     });
 
+    it('should fail jenkins artifact download if zip location is not found', function (done) {
```

Comment:
doesn't Jenkins zip the contents on the fly and return? so the pattern can be ** and not *zip*?